cmd/tailscale/cli/file.go

```diff
@@ -384,6 +387,7 @@ func openFileOrSubstitute(dir, base string, action onConflict) (*os.File, error)
 
 func receiveFile(ctx context.Context, wf apitype.WaitingFile, dir string) (targetFile string, size int64, err error) {
 	rc, size, err := tailscale.GetWaitingFile(ctx, wf.Name)
+	defer rc.Close()
```

Comment:
this panics if rc is nil on err

Reply:
thanks for the catch, moved down past the err check here.